npm/svelte/README.md

```diff
@@ -0,0 +1,83 @@
+# @cypress/svelte
+
+Mount Svelte components in the open source [Cypress.io](https://www.cypress.io/) test runner **v10.6.0+**
```

Comment:
Should be 10.7.0?

Reply:
Addressed in [b8cd5fe](https://github.com/cypress-io/cypress/pull/23553/commits/b8cd5fe1702b153d9d59a43775dffa63a5442178)